src/controller.handler/controller.handler.module.ts

```diff
@@ -33,5 +32,6 @@ import { SecretsManager } from '../profile/secrets.manager';
                 CONTROLLER_HANDLER
             ],
         };
+        return Promise.resolve(dynamicModule);
```

Comment:
This seems like the right change but (like above) I am concerned why it was in the code base for so long (and apparently working).

Reply:
My worried thought is that the tests were maybe passing trivially. Tests passing != everything is working.